app/main.py

```diff
@@ -11,3 +11,10 @@
 #     print(Actor.objects.all())
 #     Actor.objects.delete(1)
 #     print(Actor.objects.all())
+from models import Actor
+from managers import ActorManager
+
+if __name__ == "__main__":
+    Actor.objects = ActorManager()
```

Comment:
You are directly modifying the 'objects' attribute of the 'Actor' class here. This is not a recommended practice as it can lead to unexpected behavior. It would be better to instantiate 'ActorManager' in a separate variable.